src/install-opentofu.ps1

```diff
@@ -290,8 +351,19 @@ function installStandalone() {
     $dlFiles += $sumsFile
     if (!$skipVerify)
     {
-        $dlFiles += $sigFile
-        $dlFiles += $certFile
+        if ($verifyMethod -eq "cosign")
+        {
+            $dlFiles += $sigFile
+            $dlFiles += $certFile
+        } elseif ($verifyMethod -eq "gpg") {
+            if (@("1.6.0-alpha1","1.6.0-alpha2","1.6.0-alpha3","1.6.0-alpha4","1.6.0-alpha5","1.6.0-beta1","1.6.0-beta2","1.6.0-beta3","1.6.0-beta4","1.6.0-beta5","1.6.0-rc1").Contains($opentofuVersion)) {
+                throw [InstallRequirementNotMetException]::new("${bold}Additional tools are needed for the installation. Please read the following text carefully!${normal}`n`nOpenTofu version ${opentofuVersion} is not GPG-signed. This installer tries to verify that the OpenTofu version downloaded has been ${bold}signed by OpenTofu and has not been tampered with${normal}. This is only possible if either cosign or GPG is installed on the system. You have GPG installed, but only OpenTofu 1.6.0 and later are GPG-signed. You have the following options:`n`n1. ${bold}Install cosign${normal} and add it to your ${magenta}PATH${normal} or provide the ${magenta}-cosignPath${normal} parameter to your cosign installation.`n2. ${bold}Disable integrity verification${normal} with ${magenta}-skipVerify${normal} (${red}not recommended${normal}).")
```

Comment:
```suggestion
                throw [InstallRequirementNotMetException]::new("${bold}Additional tools are needed for the installation. Please read the following text carefully!${normal}`n`nOpenTofu version ${opentofuVersion} is not GPG-signed. This installer tries to verify that the OpenTofu version downloaded has been ${bold}signed by OpenTofu and has not been tampered with${normal}. This is only possible if either cosign or GPG is installed on the system. You have GPG installed, but only OpenTofu 1.6.0 and later are GPG-signed. You have the following options:`n`n1. ${bold}Install cosign${normal} and add it to your ${magenta}PATH${normal} or provide the ${magenta}-cosignPath${normal} parameter to your cosign installation.`n2. ${bold}Disabling integrity verification${normal} with ${magenta}-skipVerify${normal} (${red}not recommended${normal}).")
```

Reply:
Thank you for this, we are moving the Powershell changes to another PR, I'll add your changes there.